hunts-api/routes/hunts.js

```diff
@@ -33,4 +33,12 @@ router.post('/', (req,res) => {
   }
 });
 
+server.get('/:id', (req,res) =>{
+  var hunt = db.collection('hunts')
+  .findOne({
+    _id: req.params.id
+  });
```

Comment:
findOne takes a callback, and must use the ObjectID around the id passed: https://stackoverflow.com/questions/8233014/how-do-i-search-for-an-object-by-its-objectid-in-the-console

Reply:
This fails with ObjectId is not found. You need to require it above. `const ObjectId = require('mongodb').ObjectID;`